tck/src/main/java/com/github/bsideup/liiklus/records/RecordStorageTestSupport.java

```diff
@@ -87,7 +87,10 @@
             Supplier<CompletionStage<Map<Integer, Long>>> offsetsProvider
     ) {
         return Flux.from(getTarget().subscribe(getTopic(), UUID.randomUUID().toString(), offsetReset).getPublisher(offsetsProvider))
-                .flatMapIterable(it -> it::iterator)
+                .flatMapIterable(it -> {
+                    var iterator = it.iterator();
+                    return () -> iterator;
```

Comment:
can you give some insights on this expansion?

Reply:
there was [a change in Reactor](https://github.com/reactor/reactor-core/pull/2021/files#diff-8145aa4a75c3c22c25d5e94ab94426efR56) that made it call `Iterable#iterator` two times (completely valid) but, in case of Streams, one can call `Iterable#iterator` only once, hence the change :)